imagepipeline-base/src/main/java/com/facebook/imagepipeline/common/ImageDecodeOptionsBuilder.java

```diff
@@ -197,26 +200,6 @@ public ImageDecodeOptionsBuilder setBitmapConfig(Bitmap.Config bitmapConfig) {
     return this;
   }
 
-  /**
-   * Gets whether to allow or not an image color space to be transformed into sRGB.
-   *
-   * @return whether to allow the color space to be transformed into sRGB.
-   */
-  public boolean getTransformToSRGB() {
-    return mTransformToSRGB;
-  }
-
-  /**
-   * Sets whether to allow or not the color space transformation of the image.
-   *
-   * @param transformToSRGB whether to allow the color space to be transformed to sRGB
-   * @return this builder
-   */
-  public ImageDecodeOptionsBuilder setTransformToSRGB(boolean transformToSRGB) {
```

Comment:
In order to not break the API, we could deprecate this method and just set the color space accordingly (if we decide to support the `null` variant. Otherwise, this will be a no-op I suppose and we can just remove it.

Reply:
To make sure this library provides benefit of helping manage color, I would suggest removing this, and don't support the `null` variant.